docs/changes/0.14.0.rst

```diff
@@ -0,0 +1,43 @@
+Changelog for QCoDeS 0.14.0
+===========================
+
+The May 2020 release of QCoDeS
+
+Breaking Changes:
+_________________
+
+* ..
+
+Improved drivers:
+_________________
+
+* ..
+
+Improved:
+_________
+
+* ..
+
+Documentation:
+______________
+
+* ..
+
+Deprecations:
+_____________
+
+* Community drivers that were shipped with ``QCoDeS``, and were deprecated
+  and moved to
+  `qcodes_contrib_drivers <https://github.com/QCoDeS/Qcodes_contrib_drivers>`_
+  some months ago, are now removed from ``QCoDeS``.
```

Comment:
Maybe this should be a breaking change and not a deprecation since the deprecation happened earlier?